### PR TITLE
[TF-1096] Added `CookieConsentBar` module

### PR DESCRIPTION
**Pull request by MohammadxAli**

### CODE REVIEW

File: modules/Layout/CookieConsentBar/CookieConsentBar.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                                    </p>
+                                )}
+                            </div>
+                            <div className="flex lg:justify-center lg:items-end flex-col">
+                                <div className="flex md:flex-row flex-col gap-y-4 md:gap-y-0 md:gap-x-4 mb-4">
+                                    <button
+                                        type="button"
+                                        className="border py-3 px-4 rounded-lg border-primary-dark"
+                                        onClick={onReject}
+                                    >
```
Comment (oleg-semyonov) [inline modules/Layout/CookieConsentBar/CookieConsentBar.tsx:37]: This buttons is missing any interactions styling 😕 
Also, wouldn't it be better to extend the current `Button` component to add a `secondary` variant?

**Review by oleg-semyonov — CHANGES_REQUESTED** [review]
Looks good except one part.

File: modules/Layout/CookieConsentBar/CookieConsentBar.tsx (hunk 1 above)
Comment (MohammadxAli) [reply modules/Layout/CookieConsentBar/CookieConsentBar.tsx:37]: I guess it makes more sense to add an `outlined` variant, let me come up with an implementation for that with some interaction styling and we can ask @alpower if that looks good.

**Review by oleg-semyonov — APPROVED** [review]
Looks great! 😎

**Comment by alpower** [comment]
Could we add some screenshots in the PR description when you are ready? 🙏 Would be so nice to see all your good work visually 🌟

**Comment by MohammadxAli** [comment]
> Could we add some screenshots in the PR description when you are ready? 🙏 Would be so nice to see all your good work visually 🌟

Added the preview link there, and here :)
https://theme-nextjs-marcel-preview-git-feature-tf-1096-00f93d-prezly1.vercel.app

**Review by alpower — APPROVED** [review]

**Pull request merged**